### DISCUSSION TITLE
Grove - Moisture Sensor | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#207] [Q&A]
# Grove - Moisture Sensor | Seeed Studio Wiki

Grove - Moisture Sensor

https://wiki.seeedstudio.com/Grove-Moisture_Sensor/

**Comment by Valiice** [comment]
Are there any datasheets available for this module?

**Reply by matsujirushi** [reply]
Hi,
I think the wiki and schematics should be enough.
What do you want to know?

**Reply by Valiice** [reply]
I was looking for something like [this.](https://files.seeedstudio.com/wiki/Grove-Sunlight_Sensor/res/Si1145-46-47.pdf)
The moisture sensor has a schematic and CDC file.
I already found how to read from the moisture sensor using the analog ports on the GrovePi+. Using the python library and converting it to Rust

**Reply by matsujirushi** [reply]
Hi @Valiice ,
I don't think there is any material like [this](https://files.seeedstudio.com/wiki/Grove-Sunlight_Sensor/res/Si1145-46-47.pdf).
